workshops/memory game/README.md

```diff
@@ -0,0 +1,555 @@
+---
+name: 'Memory Game'
+description: 'Create your own memory game using JavaScript'
+author: '@giridhar7632'
+image: 'https://cloud-g7cpynxc4.vercel.app/0screenshot_2020-11-11_104712.png'
+---
+
+# Memory Game
+
+Memory Game, also known as Matching Game, is a simple card game where you need to match pairs by turn over 2 cards at a time. In this workshop we are going to build a simple memory game using JavaScript. Take a peek at the final project and the code.
+
+![Final Project](https://cloud-lrj071vrh.vercel.app/0memory_game.gif)
+
+[Live Demo](https://memory-game.giridharhackclu.repl.co/) and [Source Code](https://repl.it/@Giridharhackclu/Memory-Game#index.html).
+
+## How to play...
+
+### Rules
+
+- You will start by flipping over one card
+- If the next card you flip matches, you get +1 to your score
+- These cards then disappear
+- If the next card you flip does not match, the cards flip back
+- The game continues until you match all the cards on the board
+
+## Prerequisites
+
+![prerequisites](https://cloud-h9glprsfs.vercel.app/0prerequisites.png)
+Basic knowledge of HTML5, CSS3, and JavaScript. We will use some in-built functions of JavaScript. Also the styling will be as simple as possible.
+
+## Setup
+
+[Repl.it](https://repl.it) is an online code editor. There is no need to install anything which makes coding simpler.
+
+Create a new [HTML,CSS,JS](https://repl.it/languages/html) project or you can just fork this repl [here](https://repl.it/@Giridharhackclu/Memory-game-starter#index.html). Your development environment will be ready in few seconds!
```

Comment:
If you have a starter project, you should remove the part about starting an html project. Just tell them to fork the starter project, to avoid confusion